CMake/cdat_modules/configobj_deps.cmake

```diff
@@ -0,0 +1 @@
+set(configobj_deps ${python_pkg} ${six_pkg})
```

Comment:
no numpy? no setuptools? if there's a setup.py it probably needs setuptools (distutils)